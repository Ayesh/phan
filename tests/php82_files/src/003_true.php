<?php
function test_true(true $x): true {
    return $x;
}
function test_false(false $x): false {
    return $x;
}
function test_null(null $x): null {
    return $x;
}
