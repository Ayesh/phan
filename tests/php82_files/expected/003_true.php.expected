%s:2 PhanCompatibleTrueType Cannot use true as a type before php 8.2.
%s:5 PhanCompatibleStandaloneType Cannot use false as a standalone type before php 8.2.
%s:8 PhanCompatibleStandaloneType Cannot use null as a standalone type before php 8.2.
